libcxx/utils/libcxx/test/params.py

```diff
@@ -87,6 +87,21 @@ def getStdFlag(cfg, std):
         return "-std=" + fallbacks[std]
     return None
 
+def getSpeedOptimizationFlag(cfg):
+    if _isClang(cfg) or _isAppleClang(cfg) or _isGCC(cfg):
+        return "-O3"
+    elif _isMSVC(cfg):
+        return "/O2"
+    else:
+        raise RuntimeError("Can't figure out what compiler is used in the configuration")
+
+def getSizeOptimizationFlag(cfg):
+    if _isClang(cfg) or _isAppleClang(cfg) or _isGCC(cfg):
+        return "-Os"
+    elif _isMSVC(cfg):
+        return "/O1"
```

Comment:
There seems to also be `/Os`. @mstorsjo @StephanTLavavej @CaseyCarter Do you know which is the correct one?

Reply:
`/O1` versus `/O2` are better to use than the fine-grained options.